<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mandelbrot Set Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/resources.qrc">
    <normaloff>:/images/appicon16.png</normaloff>:/images/appicon16.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuAnsicht">
    <property name="title">
     <string>Ansicht</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuAnsicht"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QLabel" name="xMinLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>x&lt;sub&gt;min&lt;/sub&gt;:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="xMinDoubleSpinBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
    </widget>
    <widget class="QLabel" name="xMaxLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>x&lt;sub&gt;max&lt;/sub&gt;:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="xMaxDoubleSpinBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
    </widget>
    <widget class="QLabel" name="yMinLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>y&lt;sub&gt;min&lt;/sub&gt;:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="yMinDoubleSpinBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
    </widget>
    <widget class="QLabel" name="yMaxLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>y&lt;sub&gt;max&lt;/sub&gt;:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="yMaxDoubleSpinBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
    </widget>
    <widget class="QPushButton" name="applyPushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionReset"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res/resources.qrc">
     <normaloff>:/images/application-exit.svgz</normaloff>:/images/application-exit.svgz</iconset>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="res/resources.qrc">
     <normaloff>:/images/help-about.svgz</normaloff>:/images/help-about.svgz</iconset>
   </property>
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="res/resources.qrc">
     <normaloff>:/images/help-aboutqt.png</normaloff>:/images/help-aboutqt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt ...</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="res/resources.qrc">
     <normaloff>:/images/zoom_in.svgz</normaloff>:/images/zoom_in.svgz</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="res/resources.qrc">
     <normaloff>:/images/zoom_out.svgz</normaloff>:/images/zoom_out.svgz</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="res/resources.qrc">
     <normaloff>:/images/zoom_original.svgz</normaloff>:/images/zoom_original.svgz</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="res/resources.qrc">
     <normaloff>:/images/filesave.png</normaloff>:/images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Export ...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
